packages/react-core/src/components/DualListSelector/DualListSelector.tsx

```diff
@@ -0,0 +1,361 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/DualListSelector/dual-list-selector';
+import { css } from '@patternfly/react-styles';
+import { Button, ButtonVariant } from '../Button';
+import AngleDoubleLeftIcon from '@patternfly/react-icons/dist/js/icons/angle-double-left-icon';
+import AngleLeftIcon from '@patternfly/react-icons/dist/js/icons/angle-left-icon';
+import AngleDoubleRightIcon from '@patternfly/react-icons/dist/js/icons/angle-double-right-icon';
+import AngleRightIcon from '@patternfly/react-icons/dist/js/icons/angle-right-icon';
+import { DualListSelectorPane } from './DualListSelectorPane';
+import { getUniqueId, PickOptional } from '../../helpers';
+
+export interface DualListSelectorProps {
+  /** Additional classes applied to the dual list selector. */
+  className?: string;
+  /** Title applied to the available options pane. */
+  availableOptionsTitle?: string;
+  /** Options to display in the available options pane. */
+  availableOptions?: React.ReactNode[];
+  /** Status message to display above the available options pane. */
+  availableOptionsStatus?: string;
+  /** Actions to be displayed above the available options pane. */
+  availableOptionsActions?: React.ReactNode[];
+  /** Title applied to the chosen options pane. */
+  chosenOptionsTitle?: string;
```

Comment:
Do we want to have a default for this or should it be required by the user to give the panes a name?

Reply:
the default for this is set on line 78. It defaults to "Chosen options" but the user can override it.

Reply:
Oh sorry, I was asking if we should just make this required instead of providing a default title. I wasn't expecting the title to come with the component by default, though maybe it's fine. wdyt @mcarrano, if you don't pass a title to either the available or chosen panes, should we provide this title by default?

Reply:
Yes, I think that's fine.